solutions/src/test/java/ru/sberbank/school/task10/FixedThreadPoolTest.java

```diff
@@ -0,0 +1,65 @@
+package ru.sberbank.school.task10;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+class FixedThreadPoolTest {
+
+    @Test
+    void workingTestRunnable() throws InterruptedException {
+        FixedThreadPool pool = new FixedThreadPool(10);
+        pool.start();
+        for (int i = 0; i < 100; i++) {
+            pool.execute(() -> {
+                try {
+                    System.out.println("Thread " + Thread.currentThread().getName() + " is working");
+                    TimeUnit.SECONDS.sleep(1);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            });
+        }
+        TimeUnit.SECONDS.sleep(15);
+    }
+
+    @Test
+    void workingTestFuture() throws InterruptedException, ExecutionException {
+        FixedThreadPool pool = new FixedThreadPool(10);
+        pool.start();
+        List<Future<String>> results = new ArrayList<>();
+        for (int i = 0; i < 100; i++) {
+            Callable<String> callable = () -> {
+                    TimeUnit.SECONDS.sleep(1);
```

Comment:
&#39;block&#39; child has incorrect indentation level 20, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-926259722>*